src/com/sap/piper/PathUtils.groovy

```diff
@@ -0,0 +1,16 @@
+package com.sap.piper
+
+class PathUtils implements Serializable {
+    static final long serialVersionUID = 1L
+
+    static String convertToAbsolutePath(def script, String path){
+        if(path && !isAbsolutePath(path)){
+            return script.env.WORKSPACE + "/" + path
```

Comment:
I would assume that depending on the environment we are running on, this will lead to problems.

Can we really assume that the workspace path is always the same?
- physical Jenkins slaves may have a different setup
- for Docker execution in Kubernetes cluster we may end up in another path since stash/unstash is used here

We cannot assume that the config resolution for e.g. the `mavenExecute` runs on the same node than the real Maven exeuction (especially in the Kubernetes case).

We should have a thorough test in a Kubernetes environment to rule out these side-effects

Reply:
I also dropped a remark earlier in that thread: I think this does not solve the issue. The issue is: we would like to adress some files regardless of surrounding `dir` statements. But `env.WORKSPACE` holds always a reference to the workspace (e.g. `/var/jenkins_home/jobs/myJob/workspace`) - not taking subfolders into account. Please cross--check if that works as expected. I would expect a `pwd` command here.

I still think addressing files with absolute pathes creates more problems than it solves since pathes relative to project root matches always, I'm not confident that the same is true for absollute pathes.